Sources/NIOSSH/Connection State Machine/Operations/AcceptsUserAuthMessages.swift

```diff
@@ -14,6 +14,14 @@
 
 protocol AcceptsUserAuthMessages {
     var userAuthStateMachine: UserAuthenticationStateMachine { get set }
+
+    var role: SSHConnectionRole { get }
+}
+
+/// This event indicates that server wants us to display the following message to the end user.
+public struct UserAuthBannerEvent: Hashable {
+    public let message: String
+    public let languageTag: String
```

Comment:
This is a bit of type design nitpicking, but for strict value types like this I would tend to recommend that we make both of these stored properties `var`. Can we also add a `public init` that allows users to construct one of these as well?